x-pack/plugins/security_solution/public/app/translations.ts

```diff
@@ -43,7 +43,7 @@ export const USERS = i18n.translate('xpack.securitySolution.navigation.users', {
 });
```

Comment:
These changes for `POLICIES` and `HOST_ISOLATION_EXCEPTIONS` are not aligned with our page titles. Is this something we have to update in our side?
cc: @kevinlog

Reply:
@dasansol92 This is what UX decided, but it does not make much sense since these links are already inside SIEM and ENDPOINTS categories in the navigation, so it is redundant information. I have a meeting with the UX team today, I will try to convince them to keep the previous titles.
cc @machadoum